apps/native-component-list/src/screens/FontScreen.tsx

```diff
@@ -241,22 +241,33 @@ function VectorIconSection() {
   ];
 
   return (
-    <Section title="vector icon to image" gap={5}>
+    <Section title="vector icon to image">
+      <Text>rendered in expo-image and RN-core Image</Text>
+      <Text>To get the pixel size of an image, multiply `renderedImage.dimension * scale`</Text>
+
       {icons.map((icon) => {
         return (
           !!icon && (
             <Fragment key={icon.uri}>
               <Text>
-                Icon rendered to image {round(icon.width)}x{round(icon.height)}
+                Icon rendered to image {round(icon.width)}x{round(icon.height)}, scale: {icon.scale}
               </Text>
-              <Image
-                source={icon}
-                style={{
-                  height: icon.height,
-                  width: icon.width,
-                  backgroundColor: 'lightgrey',
-                }}
-              />
+              <View style={{ flexDirection: 'row', gap: 4 }}>
+                <Image
+                  source={icon}
+                  style={{
+                    height: icon.height,
+                    width: icon.width,
+                    backgroundColor: 'lightgrey',
+                  }}
+                />
+                <CoreImage
+                  source={icon}
+                  style={{
+                    backgroundColor: 'lightgrey',
```

Comment:
interestingly, core image doesn't require dimensions in the style prop when they are part of `source` prop, but `expo-image` does